data/en/test_dropdown_optional.json

```diff
@@ -0,0 +1,51 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.2",
+    "survey_id": "0",
+    "title": "Dropdown Mandatory",
+    "theme": "default",
+    "legal_basis": "StatisticsOfTradeAct",
+    "groups": [{
+        "id": "dropdown",
+        "title": "Dropdown Optional",
+        "blocks": [{
+                "type": "Question",
+                "id": "dropdown-optional",
+                "questions": [{
+                    "type": "General",
+                    "id": "dropdown-optional-question",
+                    "title": "Which football team do your support?",
+                    "answers": [{
+                        "type": "Dropdown",
+                        "id": "dropdown-optional-answer",
+                        "mandatory": false,
+                        "label": "Football team",
+                        "description": "Your favourite team from the Premier League.",
+                        "options": [{
+                                "label": "Liverpool",
+                                "value": "Liverpool"
+                            },
+                            {
+                                "label": "Chelsea",
+                                "value": "Chelsea"
+                            },
+                            {
+                                "label": "Rugby is better!",
+                                "value": "Rugby is better!"
+                            },
+                            {
+                                "label": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum",
```

Comment:
We should never use 'Loren Ipsum' for anything and should use values that would simulate real world applications of the components/features we build/use.